corehq/ex-submodules/casexml/apps/case/cleanup.py

```diff
@@ -62,7 +59,6 @@ def rebuild_case_from_forms(domain, case_id, detail):
     return FormProcessorInterface(domain).hard_rebuild_case(case_id, detail)
 
 
-@unit_testing_only
```

Comment:
yea, I did think some additional check might be in order

On 20 November 2015 at 11:23, Cory Zue <notifications@github.com> wrote:

> In corehq/ex-submodules/casexml/apps/case/cleanup.py
> <https://github.com/dimagi/commcare-hq/pull/9198#discussion_r45449296>:
>
> > @@ -62,7 +59,6 @@ def rebuild_case_from_forms(domain, case_id, detail):
> >      return FormProcessorInterface(domain).hard_rebuild_case(case_id, detail)
> >
> >
> > -@unit_testing_only
>
> i did think this was good idea. maybe we could add
> i_know_what_im_doing=True as an option to override it in non-testing code
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9198/files#r45449296>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa